<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>911</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>系统维护</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>dism++</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>diskgenius</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>环境维护</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>diskgenius</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>hdtune</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>sniffer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>everything</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>260</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>自开发检查文件数Py</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>硬盘维护</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>资料维护</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>设备维护</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>生态维护</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>配置维护</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_wjj">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>40</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>441</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>C:/</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBoxlist">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>dism++</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>diskgenius</string>
       </property>
      </item>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>系统维护</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>环境维护</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>硬盘维护</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>资料维护</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>设备维护</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>生态维护</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>赞助</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>411</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>如果带哥喜欢这个软件，可以资助我点烂钱去买支激光笔射孙狗</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_10">
     <attribute name="title">
      <string>页</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>页</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
    <addaction name="actionExport"/>
    <addaction name="actionExit_2"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>窗口</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>导出配置</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>使用方法</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
